### ISSUE TITLE
Landis+Gyr E250 compliance 

### ISSUE THREAD

**Issue by sheminasalam** [DomiStyle/esphome-dlms-meter#8]
Will this work with [Landis+Gyr E250](https://www.landisgyr.com/product/e250-three-phase-direct-connected-static-energy-meter/) Net meter? It is DLMS complaint. In theory all DLMS complaint meter should be using the same protocol and hardware...right? Also this is how my meter com port is labelled. Hope some one would be able to explain how I should do the wiring also.

<img src="https://user-images.githubusercontent.com/45196177/149604194-bbdd0158-53f8-427b-a9a7-e158bd5193ab.png" width=50% height=50%>

Also can we use this [RS232 to ttl converter](https://www.aliexpress.com/item/1005002675408493.html?spm=a2g0o.detail.1000014.11.36774dd23iIceV&gps-id=pcDetailBottomMoreOtherSeller&scm=1007.40050.266918.0&scm_id=1007.40050.266918.0&scm-url=1007.40050.266918.0&pvid=9bf8e55a-8f3a-42e1-9dce-34dcbb0da569&_t=gps-id:pcDetailBottomMoreOtherSeller,scm-url:1007.40050.266918.0,pvid:9bf8e55a-8f3a-42e1-9dce-34dcbb0da569,tpp_buckets:668%232846%238111%231996&&pdp_ext_f=%7B%22sceneId%22:%2230050%22,%22sku_id%22:%2212000021659981620%22%7D&compareFields=formatted_price:Rs.%20109.40;itemId:1005002675408493;freight_formatted_price:null;source:recommend-ump;is_freeshipping:null;trade_order:7) for translation between Esp8266 and the meter

<img src="https://user-images.githubusercontent.com/45196177/149604344-25f0af5b-b634-4a8c-b020-1757770fd235.png" width=50% height=50%>

**Comment by DomiStyle** [comment]
I don't believe it will work with any other meter besides the Kaifa MA309M. Unfortunately just being DLMS compliant doesn't mean much with every energy provider implementing their own method of transmitting it.
